### ISSUE TITLE
Ambiguous use of 'init(_:)' xcode 12, swift 5

### ISSUE THREAD

**Issue by ekchhuon** [Mattijah/QKMRZScanner#15]
I got an error like this when importing the library into project.  

<img width="1440" alt="Screen Shot 2020-10-01 at 10 07 41 AM" src="https://user-images.githubusercontent.com/47171295/94761001-33139600-03ce-11eb-8081-4f05d4855b88.png">

**Comment by Mattijah** [comment]
The library nor its dependencies have been updated to work with the new Xcode/Swift/iOS. Hopefully I will finally manage to find time to have a look at it this weekend (_no promises_)

**Comment by Mattijah** [comment]
@ekchhuon Try to update the library and set version of the `SwiftyTesseract` pod to `Swift 5`. Don't forget to clean the project before compiling. Let me then please know if this helped. (I haven't yet upgraded to the latest Xcode)

![Screenshot 2020-10-04 at 23 06 59](https://user-images.githubusercontent.com/3887042/95027106-53757600-0696-11eb-8f1e-cd2fb34fd7e8.png)

**Comment by albertobitto** [comment]
I'm having the same problem as @ekchhuon, so I updated the library to 2.2.0 version (with `SwitfyTesseract` at 3.1.2, the pod was already set to `Swift 5`).
Unfortunately I'm still experiencing the problem. My setup is:

XCode 12.0.1 running on OSX 10.15.6
iPhone 8 with iOS 14.0

**Comment by Mattijah** [comment]
ok, it has something to do with the new Xcode then I guess. This is the reason why I don't update to the latest releases right away. Also, please note that this error is coming from a 3rd party library that hasn't yet made an official (new) release as far as I'm aware.

**Comment by ekam123** [comment]
It's an issue when upgrading to Xcode 12. A solution was posted over at [SwiftyTesseract](https://github.com/SwiftyTesseract/SwiftyTesseract/issues/77#issuecomment-700529717). I tried it and worked for me. 
Basically get rid of the .init and replace with Int in the BoundingBox.swift file.
```
var cgRect: CGRect {
    return CGRect(
        x: Int(originX),
        y: Int(originY),
        width: Int((widthOffset - originX)),
        height: Int((heightOffset - originY))
    )
  }
```

**Comment by albertobitto** [comment]
The solution linked by @ekam123 works for me either!

**Comment by albertobitto** [comment]
FYI, `SwiftyTesseract` relased version 3.1.3 (and also officially releasing 4.0.0, if I understand correctly) with the fix above.

**Comment by Mattijah** [comment]
The new version `3.1.3` does not seem to compile on a simulator. It only works when testing on a real device, which is not very good. I'm waiting for a reply...

**Comment by Mattijah** [comment]
Resolved in [`v2.2.1`](https://github.com/Mattijah/QKMRZScanner/releases/tag/v2.2.1). Please note, **Xcode 12** & Cocoapods >= **1.10.0.rc.1** is recommended.
